rosdep/python.yaml

```diff
@@ -964,6 +964,10 @@ python-gdown-pip:
   ubuntu:
     pip:
       packages: [gdown]
+python-genshi:
+  ubuntu:
+    trusty: [python-genshi]
+    xenial: [python-genshi]
```

Comment:
python-genshi is available on every not EOL ubuntu, please make the entry distribution agnostic:
`ubuntu: [python-genshi]`